nodestream/pipeline/extractors/stores/aws/s3_extractor.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         self.logger = getLogger(__name__)
 
     def get_object_as_io(self, key: str) -> StringIO:
-        return self.s3_client.get_object(Bucket=self.bucket, Key=key)["Body"]
+        return self.s3_client.get_object(Bucket=self.bucket, Key=key)["Body"].read().decode('utf-8')
```

Comment:
self.s3_client.get_object(Bucket=self.bucket, Key=key)["Body"] is a botocore.response.StreamingBody type,
This has a read() method that turns it into a Bytes type, and the decoding turns it into a string.

Reply:
See comment above, we do not want or need to read to a string.